Cloud/Docker/build.sh

```diff
@@ -40,7 +40,7 @@ do
             BUILD_JSC=true
             BUILD_SPIDERMONKEY=true
             BUILD_V8=true
-            BUILD_DUKTAPE=true
+            BUILD_DUKTAPE=false
```

Comment:
WDYT about renaming the option from `all` to `major`, then dropping this line entirely?